.github/workflows/distribute.yml

```diff
@@ -60,13 +60,13 @@ jobs:
       # Test executable
       - name: "Test executable, improve tool (Windows)"
         if: runner.os == 'Windows'
-        run: herbie-compiled/herbie.exe improve --threads yes bench/tutorial.fpcore improve.fpcore
+        run: herbie-compiled/herbie.exe improve --platform c-windows --threads yes bench/tutorial.fpcore improve.fpcore
```

Comment:
I think we should test that it works even without explicitly saying you're using Windows. Herbie should default to the Windows platform on Windows.